[2025-08-01T04:31:44.422+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-01T04:31:44.450+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: parquet_in_minio.download_parquet scheduled__2023-10-01T00:00:00+00:00 [queued]>
[2025-08-01T04:31:44.455+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: parquet_in_minio.download_parquet scheduled__2023-10-01T00:00:00+00:00 [queued]>
[2025-08-01T04:31:44.455+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2025-08-01T04:31:44.467+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): download_parquet> on 2023-10-01 00:00:00+00:00
[2025-08-01T04:31:44.472+0000] {standard_task_runner.py:63} INFO - Started process 1507 to run task
[2025-08-01T04:31:44.475+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'parquet_in_minio', 'download_parquet', 'scheduled__2023-10-01T00:00:00+00:00', '--job-id', '353', '--raw', '--subdir', 'DAGS_FOLDER/parquet_to_minio.py', '--cfg-path', '/tmp/tmp0wpcyd34']
[2025-08-01T04:31:44.477+0000] {standard_task_runner.py:91} INFO - Job 353: Subtask download_parquet
[2025-08-01T04:31:44.490+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-08-01T04:31:44.517+0000] {task_command.py:426} INFO - Running <TaskInstance: parquet_in_minio.download_parquet scheduled__2023-10-01T00:00:00+00:00 [running]> on host 87b19f697adc
[2025-08-01T04:31:44.589+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='parquet_in_minio' AIRFLOW_CTX_TASK_ID='download_parquet' AIRFLOW_CTX_EXECUTION_DATE='2023-10-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-01T00:00:00+00:00'
[2025-08-01T04:31:44.590+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-01T04:31:44.601+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/utils/context.py:320 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2025-08-01T04:31:48.506+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-08-01T04:31:48.507+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-01T04:31:48.519+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=parquet_in_minio, task_id=download_parquet, run_id=scheduled__2023-10-01T00:00:00+00:00, execution_date=20231001T000000, start_date=20250801T043144, end_date=20250801T043148
[2025-08-01T04:31:48.579+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-08-01T04:31:48.611+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-08-01T04:31:48.612+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2025-08-28T03:48:53.078+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-28T03:48:53.105+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: parquet_in_minio.download_parquet scheduled__2023-10-01T00:00:00+00:00 [queued]>
[2025-08-28T03:48:53.111+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: parquet_in_minio.download_parquet scheduled__2023-10-01T00:00:00+00:00 [queued]>
[2025-08-28T03:48:53.112+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2025-08-28T03:48:53.132+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): download_parquet> on 2023-10-01 00:00:00+00:00
[2025-08-28T03:48:53.136+0000] {standard_task_runner.py:63} INFO - Started process 692 to run task
[2025-08-28T03:48:53.139+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'parquet_in_minio', 'download_parquet', 'scheduled__2023-10-01T00:00:00+00:00', '--job-id', '546', '--raw', '--subdir', 'DAGS_FOLDER/parquet_to_minio.py', '--cfg-path', '/tmp/tmpujic88me']
[2025-08-28T03:48:53.141+0000] {standard_task_runner.py:91} INFO - Job 546: Subtask download_parquet
[2025-08-28T03:48:53.155+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-08-28T03:48:53.188+0000] {task_command.py:426} INFO - Running <TaskInstance: parquet_in_minio.download_parquet scheduled__2023-10-01T00:00:00+00:00 [running]> on host c3d91d0460b0
[2025-08-28T03:48:53.259+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='parquet_in_minio' AIRFLOW_CTX_TASK_ID='download_parquet' AIRFLOW_CTX_EXECUTION_DATE='2023-10-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-01T00:00:00+00:00'
[2025-08-28T03:48:53.259+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-28T03:48:53.270+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/utils/context.py:320 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2025-08-28T04:18:19.761+0000] {local_task_job_runner.py:310} WARNING - State of this instance has been externally set to restarting. Terminating instance.
[2025-08-28T04:18:19.762+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2025-08-28T04:18:19.764+0000] {process_utils.py:132} INFO - Sending 15 to group 692. PIDs of all processes in the group: [692]
[2025-08-28T04:18:19.764+0000] {process_utils.py:87} INFO - Sending the signal 15 to group 692
[2025-08-28T04:18:19.764+0000] {taskinstance.py:2611} ERROR - Received SIGTERM. Terminating subprocesses.
[2025-08-28T04:18:19.765+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-28T04:18:19.773+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/parquet_to_minio.py", line 24, in download_parquet
    for chunk in response.iter_content(chunk_size=8192):
  File "/home/airflow/.local/lib/python3.10/site-packages/requests/models.py", line 816, in generate
    yield from self.raw.stream(chunk_size, decode_content=True)
  File "/home/airflow/.local/lib/python3.10/site-packages/urllib3/response.py", line 1043, in stream
    data = self.read(amt=amt, decode_content=decode_content)
  File "/home/airflow/.local/lib/python3.10/site-packages/urllib3/response.py", line 935, in read
    data = self._raw_read(amt)
  File "/home/airflow/.local/lib/python3.10/site-packages/urllib3/response.py", line 862, in _raw_read
    data = self._fp_read(amt, read1=read1) if not fp_closed else b""
  File "/home/airflow/.local/lib/python3.10/site-packages/urllib3/response.py", line 845, in _fp_read
    return self._fp.read(amt) if amt is not None else self._fp.read()
  File "/usr/local/lib/python3.10/http/client.py", line 466, in read
    s = self.fp.read(amt)
  File "/usr/local/lib/python3.10/socket.py", line 705, in readinto
    return self._sock.recv_into(b)
  File "/usr/local/lib/python3.10/ssl.py", line 1307, in recv_into
    return self.read(nbytes, buffer)
  File "/usr/local/lib/python3.10/ssl.py", line 1163, in read
    return self._sslobj.read(len, buffer)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 2613, in signal_handler
    raise AirflowTaskTerminated("Task received SIGTERM signal")
airflow.exceptions.AirflowTaskTerminated: Task received SIGTERM signal
[2025-08-28T04:18:19.776+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=parquet_in_minio, task_id=download_parquet, run_id=scheduled__2023-10-01T00:00:00+00:00, execution_date=20231001T000000, start_date=20250828T034853, end_date=20250828T041819
[2025-08-28T04:18:19.790+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=692, status='terminated', exitcode=2, started='03:48:52') (692) terminated with exit code 2
